src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
I like the `'l'|'r'|'u'|'d'` set of options.

But to avoid confusion, I wouldn't call the attribute `orientation` (as `orientation` is already used for legends, bar ...).

Maybe `direction: 'l'|'r'|'u'|'d'` ?